bin/ream/src/main.rs

```diff
@@ -25,14 +25,16 @@ async fn main() {
 
     let main_executor = ReamExecutor::new().unwrap();
 
+    let bootnodes = Bootnodes::new();
+
     let discv5_config = discv5::ConfigBuilder::new(discv5::ListenConfig::from_ip(
         Ipv4Addr::UNSPECIFIED.into(),
         8080,
     ))
     .build();
     let binding = NetworkConfig {
         discv5_config,
-        boot_nodes_enr: vec![],
+        boot_nodes_enr: bootnodes.bootnodes,
```

Comment:
```suggestion
        bootnodes: bootnodes.bootnodes,
```

can you rename this to bootnodes